spec/classes/agent_spec.rb

```diff
@@ -23,18 +23,17 @@
     end
   end
 
-  context 'token => 1234, files_config_enabled => true' do
+  context 'token => 1234' do
     let(:facts) { FACTS }
     let(:params) {{
-      :token => '1234', :files_config_enabled => true
-    }}
+      :token => '1234'    }}
```

Comment:
This gives some sort of token....the test needs to be rewritten or scrapped (made an issue for it)

https://github.com/opsmatic/puppet-opsmatic/issues/20